schema.sql

```diff
@@ -11,6 +11,18 @@ CREATE TABLE animals2(
 
 ALTER TABLE animals ADD COLUMN species VARCHAR(30);
 
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
```

Comment:
- [ ] Please make sure `id` is an integer instead of serial.  The serial is not an SQL standard and not compatible with other SQL database systems therefore it should not be used.